infra/terraform/k8s-local/run.sh

```
@@ -67,11 +91,16 @@ create_cluster() {
 }
 
 # ─────────────────────────────────────────────
-# Re-deploy kustomize manifests
+# Re-deploy: rebuild images, reload, apply manifests
 # ─────────────────────────────────────────────
 deploy() {
+  build_images
+
   cd "$K8S_TF_DIR"
   export KUBECONFIG=$(terraform output -raw kubeconfig_path)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)